devicetypes/smartthings/testing/simulated-refrigerator-temperature-control.src/simulated-refrigerator-temperature-control.groovy

```diff
@@ -24,9 +24,9 @@ metadata {
 		command "setpointDown"
 	}
 
-	tiles {
+	tiles (scale: 1) {
```

Comment:
I think we want to stick with `scale:2` don't we? That's what we recommend for the app generally since the v2 launch.